cmd/kubectl.go

```diff
@@ -37,29 +47,80 @@ func runKubectl(ctx *cli.Context) error {
 		return err
 	}
 
-	cluster, err := getClusterByID(c, c.UserConfig.FocusedCluster())
+	config, err := loadConfig(ctx)
 	if err != nil {
 		return err
 	}
 
-	config, err := c.ManagementClient.Cluster.ActionGenerateKubeconfig(cluster)
+	currentRancherServer := config.FocusedServer()
```

Comment:
Should this function return `(server, err)` instead of just `server`? Might be a good change, but could be out of scope for this PR.

Reply:
I don't think so, the function just accesses a map.